tests/testthat/test-trace-recursive.txt

```diff
@@ -0,0 +1,16 @@
+Full:
+█
+├─eval_tidy(quo(f()))
+└─f()
+  └─g()
+
+Collapsed:
+█
+├─[ eval_tidy(...) ]
+└─f()
+  └─g()
+
+Branch:
```

Comment:
I think this view should not use the tree styling (since it's very far from the tree now)

Reply:
I like that it reminds the tree view while being sufficiently different and I find it effective at separating the backtrace from the rest of the output. If these backtraces are going to be everywhere they should be easily detectable by eye so you can ignore it and look for the error message straight away. Maybe that's possible to achieve with another kind of output? I think the branch view makes sense though.

Reply:
What if they used `-` without the `|`?